.github/workflows/build_with_conda.yml

```
@@ -0,0 +1,39 @@
+name: Build with conda
+on:
+  push:
+  pull_request:
+
+jobs:
+  unix:
+    strategy:
+      matrix:
+        include:
+          - os: macos-latest
+          - os: ubuntu-latest
+
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Install Conda environment from environment.yml
+        uses: mamba-org/provision-with-micromamba@main
+
+        with:
+          environment-file: environment_unix.yml
+          environment-name: arcticdb
+
+      - name: Build
```

We better be explicit to have new commer to this workflow understand potential failures.
```suggestion
      - name: Build ArcticDB with conda (ARCTICDB_USING_CONDA=1)
```